android/hello_sdl_android/src/main/res/layout/ui_streaming_layout.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_marginLeft="30dp"
+    android:layout_marginTop="30dp"
```

Comment:
@santhanamk this change is related more to demonstrate proposal implementation, rather than to a functional one. If it is required, I can separate this code, but, I believe, this should not go to the final release version and should be deleted. Though, it remains here right now in order to help everybody to acknowledge. Should I remove it now?